server/src/test/java/io/crate/execution/dml/upsert/TransportShardUpsertActionTest.java

```diff
@@ -193,6 +193,7 @@ public IndexResult answer(InvocationOnMock arg0) throws Throwable {
         Schemas schemas = mock(Schemas.class);
         when(tableInfo.rootColumns()).thenReturn(Collections.<Reference>emptyList());
         when(tableInfo.versionCreated()).thenReturn(Version.CURRENT);
+        when(tableInfo.ident()).thenReturn(mock(RelationName.class));
```

Comment:
Maybe worth changing this test to use SQLExecutor to create real Schemas/DocTableInfo instance instead of starting to mock simple data classes like RelationName.